### PR TITLE
[#376] migrated workflow to new generator

### PR DESCRIPTION
**Pull request by cdietrich**
[#376] migrated workflow to new generator
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.xtext.ui.tests/src-gen/org/eclipse/xtext/xtext/ui/ecore2xtext/AbstractEcore2XtextTestRuntimeModule.java [hunk 1]
```diff
@@ -1,20 +1,60 @@
 /*
- * generated by Xtext
+ * generated by Xtext unknown
```
Comment (szarnekow) [inline org.eclipse.xtext.xtext.ui.tests/src-gen/org/eclipse/xtext/xtext/ui/ecore2xtext/AbstractEcore2XtextTestRuntimeModule.java]: Meh

File: org.eclipse.xtext.xtext.ui.tests/src/org/eclipse/xtext/xtext/ui/GenerateTestLanguages.mwe2 [hunk 2]
```diff
@@ -1,63 +1,90 @@
...
+					enabled = true
 				}
+				eclipsePlugin = {
+					name = "org.eclipse.xtext.xtext.ui.tests"
+					enabled = true
+				}
+				createEclipseMetaData = true
+			}
+			code = {
+				encoding = "UTF-8"
```
Comment (szarnekow) [inline org.eclipse.xtext.xtext.ui.tests/src/org/eclipse/xtext/xtext/ui/GenerateTestLanguages.mwe2]: I thought we are using ISO in our code base?

File: org.eclipse.xtext.xtext.ui.tests/src-gen/org/eclipse/xtext/xtext/ui/ecore2xtext/AbstractEcore2XtextTestRuntimeModule.java (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.xtext.ui.tests/src-gen/org/eclipse/xtext/xtext/ui/ecore2xtext/AbstractEcore2XtextTestRuntimeModule.java]: will change

File: org.eclipse.xtext.xtext.ui.tests/src/org/eclipse/xtext/xtext/ui/GenerateTestLanguages.mwe2 (hunk 2 above)
Comment (cdietrich) [reply org.eclipse.xtext.xtext.ui.tests/src/org/eclipse/xtext/xtext/ui/GenerateTestLanguages.mwe2]: will change

File: org.eclipse.xtext.xtext.ui.tests/src-gen/org/eclipse/xtext/xtext/ui/ecore2xtext/AbstractEcore2XtextTestRuntimeModule.java (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.xtext.ui.tests/src-gen/org/eclipse/xtext/xtext/ui/ecore2xtext/AbstractEcore2XtextTestRuntimeModule.java]: @szarnekow updated the pr

**Comment by cdietrich** [comment]
@szarnekow please reexamine

**Pull request merged**